src/ray/raylet/scheduling_resources.cc

```diff
@@ -116,6 +116,12 @@ bool ResourceSet::GetResource(const std::string &resource_name, double *value) c
   return true;
 }
 
+double ResourceSet::GetNumCPUs() const {
```

Comment:
There are a couple other places in the code where we have to do this logic (I think in `node_manager.cc`, maybe elsewhere too). Can you find and replace that code with the `GetNumCPUs` method?